common/src/main/java/com/datastrato/gravitino/dto/rel/DistributionDTO.java

```diff
@@ -22,14 +23,16 @@
 public class DistributionDTO implements Distribution {
 
   public static final DistributionDTO NONE =
-      new Builder().withStrategy(Strategy.HASH).withNumber(0).withArgs(EMPTY_ARGS).build();
+      new Builder().withStrategy(Strategy.NONE).withNumber(0).withArgs(EMPTY_ARGS).build();
 
   // Distribution strategy/method
   private final Strategy strategy;
 
+  @JsonIgnoreProperties(ignoreUnknown = true)
   // Number of buckets/distribution
   private final int number;
 
+  @JsonIgnoreProperties(ignoreUnknown = true)
   private final FunctionArg[] args;
```

Comment:
What is the default value if these two fields are not not available in JSON string?

Reply:
Can this really work in json deserialization? CC @mchades

Reply:
UT should be added in `com.datastrato.gravitino.dto.rel.TestDistributionDTO`